.github/CODEOWNERS

```diff
@@ -1551,6 +1551,11 @@ x-pack/test/security_solution_cypress/cypress/tasks/expandable_flyout  @elastic/
 /x-pack/plugins/stack_connectors/server/connector_types/gemini @elastic/security-generative-ai @elastic/obs-ai-assistant
 /x-pack/plugins/stack_connectors/common/gemini @elastic/security-generative-ai @elastic/obs-ai-assistant
 
+# Inference API
+/x-pack/plugins/stack_connectors/public/connector_types/inference @elastic/response-ops
+/x-pack/plugins/stack_connectors/server/connector_types/inference @elastic/response-ops
+/x-pack/plugins/stack_connectors/common/inference @elastic/response-ops
```

Comment:
Would it make sense to have this new connector owned by @elastic/appex-ai-infra?